pkgs/by-name/ib/ibm-plex/update.sh

```diff
@@ -0,0 +1,42 @@
+#! /usr/bin/env nix-shell
+#! nix-shell -i bash -p nix-prefetch jq
+#  shellcheck shell=bash
+
+families=(
+    "serif"
+    "sans"
+    "sans-condensed"
+    "sans-arabic"
+    "sans-devanagari"
+    "sans-thai"
+    "sans-thai-looped"
+    "sans-tc"
+    "sans-kr"
+    "sans-jp"
+    "sans-hebrew"
+    "mono"
+    "math"
+)
+
+latest_release=$(curl --silent https://api.github.com/repos/IBM/plex/releases/latest)
+version=$(jq -r '.tag_name | sub("^@ibm/[\\w-]+@"; "")' <<<"$latest_release")
+
+dirname="$(dirname "$0")"
+echo \""${version}"\" >"$dirname/version-new.nix"
+if diff -q "$dirname/version-new.nix" "$dirname/version.nix"; then
+    echo No new version available, current: "$version"
+    rm "$dirname/version-new.nix"
+    exit 0
+else
+    echo Updated to version "$version"
+    mv "$dirname/version-new.nix" "$dirname/version.nix"
+fi
+
+printf '{\n' > "$dirname/hashes.nix"
```

Comment:
Just generate a JSON file and import it with `lib.importJSON` - trust me it's much simpler

Reply:
Personally, I think the current setup is technically simple enough